src/main/java/com/connexta/store/adaptors/StoreStatus.java

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright (c) 2019 Connexta, LLC
+ *
+ * Released under the GNU Lesser General Public License version 3; see
+ * https://www.gnu.org/licenses/lgpl-3.0.html
+ */
+package com.connexta.store.adaptors;
+
+public class StoreStatus {
```

Comment:
✏️ Enum

Reply:
I like the string. It keeps the Adapter non specific to Ion needs.

Reply:
The Adaptor shouldn't allow every string though. Either we limit it to an enum, or it should reject statuses that aren't one of the ones listed here.

Reply:
Only the Store cares what we tag with, why does the Adapter care?

Reply:
Let's discuss in person.